### PR TITLE
Yifan-survey-project

### PR DESCRIPTION
**Pull request by Yifan-858**
## Netlify link
https://yifan-survey-project.netlify.app/

## Collaborators
[yifan-858]
Solo

### CODE REVIEW

File: src/Components/Button/Button.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  .submit-text-area {
+    top: 160px;
+    padding: 80px;
+    font-size: 1.2rem;
+  }
+}
+
+@media screen and (max-width: 425px) {
+  .button-section {
```
Comment (kathinka) [inline src/Components/Button/Button.css:41]: Not so responsive on mobile or pad - the nav buttons is covering the options on the checkbox list for instance. maybe assign them to a row in your grid thats beneath the questions?

File: src/Components/ColorType/ColorType.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  align-items: center;
+}
+
+.color-option-area p {
+  font-size: var(--option-font-size);
+  font-weight: var(--option-font-weight);
+  margin: 20px auto;
+}
+
+.color-wheel {
```
Comment (kathinka) [inline src/Components/ColorType/ColorType.css:14]: I could not close the picker only using the keyboard, so maybe use a different one?  Maybe use the https://jscolor.com/ instead? - that one is possible to close just with enter :) if you are aiming for accessibility max  though!

but cool to see the colorpicker in use!

File: src/Data/questionList.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
+{
+  "questions": [
```
Comment (kathinka) [inline src/Data/questionList.json:2]: Clever to make a Json for all the questions and options!

File: src/Components/DropdownType/DropdownType.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  position: relative;
+}
+
+.dropdown-area {
+  position: absolute;
+  top: 100%;
+  width: 100%;
+  overflow: hidden;
+  max-height: 0;
+  transition: max-height 0.2s ease-in-out;
```
Comment (kathinka) [inline src/Components/DropdownType/DropdownType.css:31]: nice touch with a transition here :)

File: src/Components/DropdownType/DropdownType.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  cursor: pointer;
+  text-align: center;
+  margin-top: -1px;
+  padding: 5px;
+  border: var(--main-color) 1px solid;
+  border-radius: 30px;
+  color: var(--main-color);
+  font-size: var(--option-font-size);
+  font-weight: var(--option-font-weight);
+  transition: all 0.2s ease-in-out;
```
Comment (kathinka) [inline src/Components/DropdownType/DropdownType.css:48]: maybe set background-color instead of all on the transition here :)

**Review by kathinka — COMMENTED** [review]
Hi Yifan! 
Really great work this week 🥇 -  you managed to make a site that got a 100% accessible score, and you included a lot of different input choices!  So cool to see the color picker in use :)
I also liked the possibility to go back and forth in the form, and that each question was rendered on its own. 
 Really nice and clean code, easy to read with all the parts of the form (component and css) separated in different folders. 🥇 
 I sure got a lot of ideas for improvements in my own project!
 
 You already passed with more than the basics, so its challenging to find something to "pick on" but I found some small possibilities for adjustments (I left a comment where its applicable) so you could concider if you would polish this app further :)
 
 Really great work - I will have to do some serious work on my owen project to keep up with your standards! 💯

**Comment by Yifan-858** [comment]
> Hi Yifan! Really great work this week 🥇 - you managed to make a site that got a 100% accessible score, and you included a lot of different input choices!  So cool to see the color picker in use :) I also liked the possibility to go back and forth in the form, and that each question was rendered on its own.  Really nice and clean code, easy to read with all the parts of the form (component and css) separated in different folders. 🥇  I sure got a lot of ideas for improvements in my own project!    You already passed with more than the basics, so its challenging to find something to "pick on" but I found some small possibilities for adjustments (I left a comment where its applicable) so you could concider if you would polish this app further :)    Really great work - I will have to do some serious work on my owen project to keep up with your standards! 💯

Thank you so much Kathinka for the review!❤️  I haven't noticed there are so many things I missed. I found it challenging to keep the CSS files nice and clean this time since there are so many pages. But I will improve as much as I can!

**Review by AntonellaMorittu — APPROVED** [review]
Hello Yifan and congrats with your Survey project! The app satisfies the requirements and you even implemented the multi-step survey! Great! Well done with responsiveness and accessibility too. Love the color-picker! ❤️ 
A tip to improve UX is to prevent the user to skip to the next question if the current one is unanswered. Another thing to update is the name of `components` and `data` folder to be lowercase. Anyway, you did a great job! 👏

**Pull request closed without merging**